packages/vaadin-themable-mixin/vaadin-theme-property-mixin.js

```diff
@@ -25,21 +25,42 @@ export const ThemePropertyMixin = (superClass) =>
          * See [Styling Components: Sub-components](https://vaadin.com/docs/latest/ds/customization/styling-components/#sub-components).
          * page for more information.
          *
+         * @deprecated The `theme` property is not supposed for public use and will be dropped in Vaadin 24.
```

Comment:
`vaadin-theme-property-mixin.d.ts` should be updated accordingly

Reply:
Done. https://github.com/vaadin/web-components/pull/3529/commits/d0b0ccc3c1edfa7f1487099ad9ce13311e6e991e